motoko/threshold-schnorr/README.md

```diff
@@ -0,0 +1,276 @@
+---
+keywords: [advanced, motoko, threshold schnorr, schnorr, signature]
+---
+
+# Threshold Schnorr
+
+[View this sample's code on GitHub](https://github.com/dfinity/examples/tree/master/motoko/threshold-schnorr)
+
+## Overview
+
+We present a minimal example canister smart contract for showcasing the
+[threshold
+Schnorr](https://internetcomputer.org/docs/current/developer-docs/integrations/t-schnorr)
+API. TODO: fix link when the API docs are published.
```

Comment:
But none of those can be resolved currently, right?

Reply:
All TODOs seem more like they could/should be tracked in a CRP-ticket, because they are TODOs for the Crypto team, and not for the reader of the README.

Optionally, related to the first TODO, for the API, you could point to
* https://org5p-7iaaa-aaaak-qckna-cai.icp0.io/docs#ic-schnorr_public_key and
* https://org5p-7iaaa-aaaak-qckna-cai.icp0.io/docs#ic-sign_with_schnorr

which is the [preview](https://github.com/dfinity/interface-spec/pull/288#issuecomment-2019103049) of https://github.com/dfinity/interface-spec/pull/288.

Reply:
Created a ticket, added the ticket id to the READMEs, and added a link to the preview.

Reply:
Thanks. Having the TODOs only in the JIRA ticket (and removing them here) would also have been OK IMO.

Reply:
I removed the TODOs from READMEs and added some those to the ticket.